#!/usr/bin/python3

def raise_exception_msg(message="Good Work"):
    raise NameError(message)